From 9e28214eb784b9f68af6e0503f8cefe861f13440 Mon Sep 17 00:00:00 2001
From: Shesha Sreenivasamurthy <shesha@cisco.com>
Date: Wed, 2 Sep 2015 08:55:43 -0700
Subject: [PATCH 5/9] eal: Temporarily turn off unthrottled RTE_LOG(...)

Otherwise, /var/log/syslog eventually fills the disk. The error
condition seems only to affect ESXi VM's. It'd be worth suggesting log
throttling to the DPDK community. Much better to avoid making syslog
(...) calls in the first place.
---
 lib/librte_eal/linuxapp/eal/eal_interrupts.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/librte_eal/linuxapp/eal/eal_interrupts.c b/lib/librte_eal/linuxapp/eal/eal_interrupts.c
index 3f87875..29a3539 100644
--- a/lib/librte_eal/linuxapp/eal/eal_interrupts.c
+++ b/lib/librte_eal/linuxapp/eal/eal_interrupts.c
@@ -709,7 +709,7 @@ eal_intr_process_interrupts(struct epoll_event *events, int nfds)
 		 * for epoll_wait.
 		 */
 		bytes_read = read(events[n].data.fd, &buf, bytes_read);
-		if (bytes_read < 0) {
+		if (0 && bytes_read < 0) {
 			if (errno == EINTR || errno == EWOULDBLOCK)
 				continue;
 
-- 
2.5.0

